### DISCUSSION TITLE
ioQuake3DS build  with devkitarm

### DISCUSSION THREAD

**Discussion by davidpj46** [profi200/open_agb_firm#194] [General]
Hi,

I am trying to build ioQuake3DS-1.2.0 with devkitarm and I am getting an error
qcommon/common.c :28:10: fatal error: netinet/in.h : no such file or directory

this error was not occuring a few weeks ago and I could compile with no issues.

Any ideas of what might be happening?

**Comment by profi200** [comment]
You are probably in the wrong repository to ask about building Quake for 3DS.
